definitions/infra-azureappliactioninsight/summary_metrics.yml

```diff
@@ -0,0 +1,17 @@
+providerAccountName:
```

Comment:
Minor typo on the folder name here: s/infra-azureappliactioninsight/infra-azureapplicationinsight/g